include/correction.h

```diff
@@ -35,6 +36,7 @@ typedef std::variant<double, Binning, MultiBinning, Category, Formula> Content;
 class Formula {
   public:
     enum class ParserType {TFormula, numexpr};
+    static bool eager_compilation; // true by default
```

Comment:
Not thread safe

Reply:
Thanks for the review! Indeed I was thinking about this, but my reasoning is that whatever state this is in when the Formula is constructed decides the eagerness of that Formula object, and regardless of its value, the Formula will compile itself eventually. Basically some race condition might mean that some Formulas will be lazy when they were not desired, but the end result is harmless.

Reply:
That is against CMS threading policy. The C++ standard says their are not guarnteed 'safe' race conditions and everything else is undefined behavior. This should become a std::atomic if you want this behavior. Better yet, remove the static and make laziness a parameter passed to the constructor.

Reply:
Ok, I mean it's two states, but undefined behavior is bad. I'll toss an atomic wrapper around it. The constructor is meant to take the data format, which shouldn't be concerned with evaluation.